ajax/libs/bulmaswatch/package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "bulmaswatch",
+    "version": "0.4.1",
+    "description": "Themes for Bulma",
+    "author": "Jenil Gogari <i@jgog.in>",
+    "homepage": "https://jenil.github.io/bulmaswatch",
+    "filename": "default/bulmaswatch.min.css",
```

Comment:
@sufuf3 I'm not sure if using the `basePath` is apt since each bulmaswatch theme has it's own directory, like `default`, `flatly`, etc.